kolibri/plugins/coach/assets/src/views/channel-list-page/index.vue

```diff
@@ -49,8 +49,13 @@
     },
     methods: {
       reportLink(channelId) {
+        const linkTargets = {
+          [PageNames.RECENT_CHANNELS]: PageNames.RECENT_ITEMS_FOR_CHANNEL,
+          [PageNames.TOPIC_CHANNELS]: PageNames.TOPIC_CHANNEL_ROOT,
+          [PageNames.LEARNER_CHANNELS]: PageNames.LEARNER_CHANNEL_ROOT,
+        };
         return {
-          name: PageNames.RECENT_ITEMS_FOR_CHANNEL,
+          name: linkTargets[this.pageName],
           params: {
```

Comment:
don't we need to send in more params than this? like `topicId` and stuff?

I did it another way in my branch, but this is cleaner, I think